chore: fix blog typo (#4920)
CloudFire-LLC/cloudfire: website/src/app/blog/how-dns-works-in-firezone/readme.mdx

1. An admin defines a DNS Resource with address `*.slack.com` in the Firezone
   admin portal. Notice the wildcard -- this will route all subdomains for Slack

>    through Firezone as well, which helps ensure all requests are routed.

Author: ```suggestion
   through Firezone as well, which helps ensure all relevant Slack traffic is routed.
```
